src/sage/doctest/parsing.py

```diff
@@ -587,6 +587,23 @@ def parse(self, string, *args):
             sage: print(m)
             87654321
 
+        Optional tag at the start of an example block persists to the end of
+        the block::
+
```

Comment:
Need to clarify that a blank line ends the block in this sense even though for Sphinx, the block continues